### ISSUE TITLE
A question about loss_repeat in training code

### ISSUE THREAD

**Issue by AndysonYs** [baaivision/NOVA#8]
Thanks for your excellent works and the training guide.
I've noticed there is a hyper-param called loss_repeat in the config file. May i ask that why did you repeat the loss computing for 4 times in the T2V training?

**Comment by PhyscalX** [comment]
1.  ``RepeatSampling`` trick was adopted in [MAE-ST](https://arxiv.org/abs/2205.09113), which enlarges the *effective* batch size.
2. ``RepeatSampling`` trick was also adopted in MAR, but not mentioned in the paper. See this [issue](https://github.com/LTH14/mar/issues/14).

**Comment by AndysonYs** [comment]
Got it. Thanks!
